build.gradle

```diff
@@ -13,8 +13,8 @@ buildscript {
 	}
 
 	dependencies {
-		classpath 'io.spring.gradle:spring-build-conventions:0.0.27.RELEASE'
-		classpath "org.springframework.boot:spring-boot-gradle-plugin:$springBootVersion"
+		classpath 'io.spring.gradle:spring-build-conventions:0.0.34.RELEASE'
```

Comment:
In order to do this update we will need to incorporate the changes from d8f160c178d86d2421ba3f8bfba8ea183af436cb and 5acb307a54c7a5347ffce8b4be7adb9bd3339aa5.